### ISSUE TITLE
nuget segfaults during builds

### ISSUE THREAD

**Issue by SirCmpwn** [travis-ci/travis-ci#4044]
cc @joshua-anderson @akoeplinger @nterry

Here's an example build that fails: https://travis-ci.org/SirCmpwn/TrueCraft/builds/65771410

I had similar problems locally for a while, but an upgrade to mono and nuget resolved the issue.

**Comment by akoeplinger** [comment]
Hmm, I only see it once in your build logs, did it happen more often? Anyways, this is a Mono bug and should be reported at https://bugzilla.xamarin.com (preferably with repro steps).

**Comment by SirCmpwn** [comment]
Yeah, it happens for all of my builds now. And it was reported on bugzilla, and fixed, and the solution is "upgrade mono and nuget".

**Comment by akoeplinger** [comment]
Your [build list](https://travis-ci.org/SirCmpwn/TrueCraft/builds) seems pretty green to me, can you link to some more failed builds? You're already running latest Mono, maybe it was fixed in a commit that didn't make it into a release yet, can you link me to the bugzilla thread?

**Comment by SirCmpwn** [comment]
See this instead: https://travis-ci.org/SirCmpwn/TrueCraft/pull_requests

I don't have the bugzilla thread handy, you'll have as much success with googling it as I would.

**Comment by akoeplinger** [comment]
The [first red build](https://travis-ci.org/SirCmpwn/TrueCraft/builds/65771410) is the one you linked to in the initial post and the only [other red build](https://travis-ci.org/SirCmpwn/TrueCraft/builds/65790350) fails due to some unrelated git checkout problems.

**Comment by SirCmpwn** [comment]
I feel like there were more failures than that. I recall at least two. Oh well.

**Comment by Joshua-Anderson** [comment]
@SirCmpwn Have you seen this with the latest mono lately?

**Comment by SirCmpwn** [comment]
I haven't been paying attention, and my mono workload for travis has been pretty light for a couple of weeks. I'll start paying attention and update this issue when I reach a conclusion.
